code/modules/mob/living/simple_animal/bot/bot.dm

```diff
@@ -75,14 +75,21 @@
 	var/control_freq = BOT_FREQ		// bot control frequency
 	var/bot_filter 				// The radio filter the bot uses to identify itself on the network.
 	var/bot_type = 0 //The type of bot it is, for radio control.
+	var/data_hud_type = DATA_HUD_DIAGNOSTIC //The type of data HUD the bot uses. Diagnostic by default.
 	//This holds text for what the bot is mode doing, reported on the remote bot control interface.
 	var/list/mode_name = list("In Pursuit","Preparing to Arrest", "Arresting", \
 	"Beginning Patrol", "Patrolling", "Summoned by PDA", \
 	"Cleaning", "Repairing", "Proceeding to work site", "Healing", \
 	"Responding", "Navigating to Delivery Location", "Navigating to Home", \
 	"Waiting for clear path", "Calculating navigation path", "Pinging beacon network", "Unable to reach destination")
 
-	hud_possible = list(DIAG_STAT_HUD, DIAG_BOT_HUD, DIAG_HUD) //Diagnostic HUD views
+	var/datum/atom_hud/data/bot_path/path_hud = new /datum/atom_hud/data/bot_path()
+	var/path_image_icon = 'icons/obj/aibots.dmi'
+	var/path_image_icon_state = "path_indicator"
+	var/path_image_color = "#FFFFFF"
+	var/reset_access_timer_id
+
+	hud_possible = list(DIAG_STAT_HUD, DIAG_BOT_HUD, DIAG_HUD, DIAG_PATH_HUD = HUD_LIST_LIST)//Diagnostic HUD views
```

Comment:
What does `DIAG_PATH_HUD = HUD_LIST_LIST` do...?

Reply:
i am looking at this right i think it has to do with the path images being created
//by default everything in the hud_list of an atom is an image
//a value in hud_list with one of these will change that behavior
 #define HUD_LIST_LIST 1